job/reminders/reminderPoller.js

```diff
@@ -1,20 +1,28 @@
 /* eslint-disable no-await-in-loop */
-module.exports = (db, incidentClient, sendReminder, eventPublisher, emailResolver) => {
+const logger = require('../../log')
+
+module.exports = (db, incidentClient, reminderSender, eventPublisher, emailResolver) => {
   const processReminder = async client => {
     const reminder = await incidentClient.getNextNotificationReminder(client)
-
+    logger.info('Found reminder reminder', reminder)
```

Comment:
Is this meant to say `reminder reminder`? 😄

Reply:
no - fixed this and refactored this method a bit